api/models/groups.py

```diff
@@ -0,0 +1,82 @@
+from django.db import models
+
+from api.models.base import GroupOrPerson
+from api.models.constants import DEFAULT_CHOICE_CHAR_FIELD_MAX_LENGTH
+from api.models.days_time_slots import DayAndTimeSlot
+from api.models.languages_levels import Language, LanguageAndLevel
+from api.models.people import Coordinator, Student, Teacher, TeacherUnder18
+
+
+class GroupCommon(GroupOrPerson):
+    """Abstract model for attributes shared by regular groups and speaking clubs."""
+
+    coordinators = models.ManyToManyField(Coordinator)
+    students = models.ManyToManyField(Student)
+    teachers = models.ManyToManyField(Teacher)
+    # group chat created manually by the coordinator/teacher
+    telegram_chat_url = models.URLField(null=True, blank=True)
+
+    class Meta:
+        abstract = True
+
+
+# TODO think about group clusters for auto-constructed groups
+class Group(GroupCommon):
+    """Model for a regular language group."""
+
+    class Status(models.TextChoices):
+        PENDING = "pending", "Pending"
+        STUDYING = "study", "Studying"
+        FINISHED = "finish", "Finished"
+        # TODO put statuses here once they are finalized
+
+    availability_slot = models.ManyToManyField(DayAndTimeSlot)
+    is_for_staff_only = models.BooleanField(default=False)
+    language_and_level = models.ForeignKey(LanguageAndLevel, on_delete=models.PROTECT)
+    lesson_duration = models.PositiveSmallIntegerField()
```

Comment:
is it clear that I mean minutes? Maybe add `verbose_text`?

Reply:
I suggest to write it in field title, kind of `lesson_duration_min` or `lesson_duration_in_minutes` to not mix up with "minimum"

Reply:
Done